packages/protocol/contracts/governance/Validators.sol

```diff
@@ -324,6 +325,51 @@ contract Validators is
     return (epochs, membershipGroups, history.lastRemovedFromGroupTimestamp);
   }
 
+  /**
+   * @notice Calculates the validator score for an epoch from the uptime value for the epoch.
+   * @param uptime The Fixidity representation of the validator's uptime, between 0 and 1.
+   * @dev epoxh_score = uptime ** exponent
+   * @return Numerator and denominator of the epoch score.
+   */
+  function calculateEpochScore(uint256 uptime) public view returns (uint256, uint256) {
```

Comment:
Consider having this return the fixidity representation of the uptime score

Reply:
Done